qemu/tests/cfg/nvdimm.cfg

```diff
@@ -26,9 +29,7 @@
     mount_dir = "/mnt"
     pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=1024"
     dev_path = /dev/pmem0
-    format_command = "mkfs.xfs -f ${dev_path}"
-    RHEL.8:
-        format_command += " -m reflink=0"
+    format_command = "mkfs.xfs -m reflink=0 -f ${dev_path}"
```

Comment:
Have we stopped supporting rhel7 at all ?

Reply:
Thank you for the correction, I originally thought `reflink` exists on all xfs versions, but the fact is negative.
```
       -m global_metadata_options
              These options specify metadata format options that  either  apply  to
              the  entire  filesystem  or aren't easily characterised by a specific
              functionality group. The valid global_metadata_options are:

                   crc=value
                          This is used to create a filesystem which  maintains  and
                          checks  CRC  information in all metadata objects on disk.
                          The value is either 0 to disable the  feature,  or  1  to
                          enable the use of CRCs.

                          CRCs  enable  enhanced  error  detection  due to hardware
                          issues, whilst the format  changes  also  improves  crash
                          recovery  algorithms  and the ability of various tools to
                          validate and repair metadata corruptions  when  they  are
                          found.  The CRC algorithm used is CRC32c, so the overhead
                          is dependent on CPU architecture as some CPUs have  hard‐
                          ware acceleration of this algorithm.  Typically the over‐
                          head of calculating and checking the CRCs is not  notice‐
                          able in normal operation.

                          By default, mkfs.xfs will enable metadata CRCs.

                   finobt=value
                          This  option  enables  the  use  of a separate free inode
                          btree index in each allocation group. The value is either
                          0  to  disable  the  feature, or 1 to create a free inode
                          btree in each allocation group.

                          The free inode btree mirrors the existing allocated inode
                          btree  index which indexes both used and free inodes. The
                          free inode btree does not  index  used  inodes,  allowing
                          faster,  more  consistent inode allocation performance as
                          filesystems age.

                          By default, mkfs.xfs will not create  free  inode  btrees
                          for  filesystems.   This  is  for backwards compatibility
                          with older RHEL7 kernels.  If the  free  inode  btree  is
                          enabled,  older  RHEL7  kernels will not be able to mount
                          the created filesystem.  When  the  option  -m  crc=0  is
                          used,  the  free inode btree feature is not supported and
                          is disabled.

                   uuid=value
                          Use the given value as the filesystem UUID for the  newly
                          created  filesystem.  The default is to generate a random
                          UUID.

```